spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'bank_account'
+
+describe Bank::Account do
+  let(:account) {Bank::Account.new("check_it", 1000)}
+
+  describe "attributes" do
+    it "responds to :balance" do
+      expect(account).to respond_to :balance
+    end
+
+    it "responds to :id" do
+      expect(account).to respond_to :id
+    end
+
+    it "has the id 'checkit'" do
+      expect(account.id).to eq("check_it")
+    end
+
+    it "has an initial balance of 1000" do
+      expect(account.balance).to eq(1000)
+    end
+
+    it "returns 750 for a 250 withdrawal" do
+      expect(account.withdraw(250)).to eq(750)
+    end
```

Comment:
For this test and the ones on lines 15 and 17, I'd recommend making your it statement a little more descriptive of the general behavior you're testing rather than what you expect from your test data. For instance, instead of:
```ruby 
it "returns 750 for a 250 withdrawal" do
```
you could do:
```ruby 
it "withdraws the specified amount" do
```
It makes it easier for someone skimming your tests to see what kinds of behaviors they can expect from your class. It'll also be helpful for you to remind yourself what the behavior should be as your projects get bigger. Also, more general, behavior-based statements mean you'll have to do less clean up if you ever decide to change your test data.